tests/README.md

```
@@ -0,0 +1,182 @@
+# Tests
+
+This directory contains the test suite for svelte-rune-highlight.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)